octokit/pagination.py

```diff
@@ -0,0 +1,38 @@
+from .resources import Resource
+
+class Pagination(object):
+  def __init__(self, *args, **kwargs):
+    # TODO (howei): possibly extract auto_paginate from kwargs
+    # so users can do client = Client(auto_paginate=True)
+    self.auto_paginate = False
+    self.last_response = None
```

Comment:
I think last reponse makes more sense in Client.. pagination doesn't use it, and other mixins do (like rate_limit)